testing/test_mercurial.py

```diff
@@ -2,7 +2,15 @@
 from setuptools_scm.hg import archival_to_version, parse
 from setuptools_scm import integration
 from setuptools_scm.config import Configuration
+from setuptools_scm.utils import has_command
 import pytest
+import warnings
+
+
+with warnings.catch_warnings():
+    warnings.filterwarnings('ignore')
```

Comment:
Note that I hate this but I have no clue how to otherwise make pytest not crash about `has_command()` throwing a warning. Nor that I have any clue why a command returning a bool would issue a warning that it returns False.